lib/kdump_utils.pm

```diff
@@ -351,7 +351,7 @@ sub check_function {
 
     if ($args{test_type} eq 'function') {
         # Check, that vmcore exists, otherwise fail
-        assert_script_run('ls -lah /var/crash/*/vmcore');
+        assert_script_run('ls -lah /var/crash/*/vmcore', 120);
```

Comment:
default is 90s, maybe to be more in the safe side, shall we double the default? so `180`?